community/modules/scheduler/schedmd-slurm-gcp-v6-controller/controller.tf

```diff
@@ -83,23 +84,88 @@ module "slurm_controller_template" {
 }
 
 # INSTANCE
+locals {
+  # TODO: add support for proper access_config
+  access_config = {
+    nat_ip       = null
+    network_tier = "STANDARD"
+  }
+}
+
 module "slurm_controller_instance" {
-  source = "github.com/SchedMD/slurm-gcp.git//terraform/slurm_cluster/modules/slurm_controller_instance?ref=6.2.0"
+  source = "github.com/SchedMD/slurm-gcp.git//terraform/slurm_cluster/modules/_slurm_instance?ref=6.2.0"
+
+  access_config       = !var.disable_controller_public_ips ? [local.access_config] : []
+  add_hostname_suffix = false
+  hostname            = "${local.slurm_cluster_name}-controller"
+  instance_template   = local.have_template ? var.instance_template : module.slurm_controller_template[0].self_link
+
+  project_id          = var.project_id
+  region              = var.region
+  slurm_cluster_name  = local.slurm_cluster_name
+  slurm_instance_role = "controller"
+  static_ips          = var.static_ips
+  subnetwork_project  = var.subnetwork_project
+  subnetwork          = var.subnetwork_self_link
+  zone                = var.zone
+
+  metadata = var.metadata
+
+  depends_on = [
+    module.slurm_files,
+    # Ensure nodes are destroyed before controller is
+    module.cleanup_compute_nodes[0],
+  ]
+}
+
+# SECRETS: CLOUDSQL
+resource "google_secret_manager_secret" "cloudsql" {
+  count = var.cloudsql != null ? 1 : 0
+
+  secret_id = "${local.slurm_cluster_name}-slurm-secret-cloudsql"
+
+  replication {
```

Comment:
This syntax is deprecated and will break in TPG 5.0. It likely generates warnings you may not have noticed. Please adopt the syntax from 1d3a2b09